server/src/main/java/io/spine/server/entity/EntityClass.java

```diff
@@ -97,8 +98,13 @@ private static ModelError noSuchConstructor(String entityClass, String idClass)
 
     /**
      * Retrieves the state class of the passed entity class.
+     *
+     * <p>Method is public because another way to get state class is create {@code EntityClass}
```

Comment:
This is too much information for a user who just needs to understand what this method is for.

This is enough:
```
<p>Though this method is {@code public}, it is <em>not</em> considered a part of the public API. It is used internally by other framework routines and not designed for efficient execution by Spine users.
```